packages/cfd/src/sass/cfd-dashboard.scss

```diff
@@ -504,66 +507,54 @@
 
         &--pending {
             background-color: #ffad3a;
-            color: #333333;
             border-radius: 0.5rem;
         }
 
-        &--verified {
+        &--poi_verified {
+            background: #377cfc;
```

Comment:
This needs to be reused from `Theme.scss`,
because we already have this color code in constants.scss